readme.md

```diff
@@ -108,6 +109,7 @@ Currently, these are the flags you can configure:
 - `yarn` - Use yarn if possible (`true` by default).
 - `contents` - Subdirectory to publish (`.` by default).
 - `releaseDraft` - Open a GitHub release draft after releasing (`true` by default).
+- `2Fa` - Enable 2FA on new packages (`true` by default)
```

Comment:
```suggestion
- `2fa` - Enable 2FA on new packages (`true` by default)
```

Reply:
This will require some code change. The CLI code converts `--no-2fa` to `{ "2FA": false }` (yes, including the capitalized `F`), which is in turn passed to the `ui.js` module as an option.

If you want the option to be named `2fa` (without the capitalization), we'll have to normalize it at some point (ideally normalizing the CLI to `2fa` instead of `2FA`)

Reply:
Yes, let's do that. `2Fa` is really strange. Ideally, `meow` would have some kind of transform option for flags, but it doesn't.